src/main/resources/db/migration/postgresql/V6.0__drop_resource_state_per_negotiation_table.sql

```diff
@@ -0,0 +1 @@
+drop table if exists public.resource_state_per_negotiation;
```

Comment:
please add the new constraint you have created for Negotiation.sourceID

Reply:
Fixed in 41ce33a45006cdba6c6bd86db29045475b6c2e00